Symfony/CS/ConfigurationException/InvalidConfigurationException.php

```diff
@@ -24,9 +24,10 @@ class InvalidConfigurationException extends \InvalidArgumentException
 {
     /**
      * @param string $message
+     * @param int    $code
      */
-    public function __construct($message)
+    public function __construct($message, $code = FixCommand::EXIT_STATUS_FLAG_HAS_INVALID_CONFIG)
```

Comment:
that would not only check against `null` right? (like `0` would become the default as well?)